test/bugs-reported/bug7.chpl

```diff
@@ -0,0 +1,15 @@
+use Graph;
```

Comment:
I think this would be more appropriate as a JIRA issue that included a description of the kinds of graph representations you would like to see.